src/android/io/branch/BranchSDK.java

```diff
@@ -271,7 +266,7 @@ private void redeemRewards(final int value, CallbackContext callbackContext)
 
         Log.d(LCAT, "start redeemRewards()");
 
-        this.instance.redeemRewards(value, new LoadRewardsListener(callbackContext));
+        this.instance.redeemRewards(value, new RedeemRewardsListener(callbackContext));
```

Comment:
LoadRewardsListener probably returned the wrong value here, so I created a RedeemRewardsListener.